test_build_bundle_image.sh

```diff
@@ -15,19 +15,26 @@ function main {
 function set_up {
 	common_set_up
 
+	export LATEST_TOMCAT_VERSION_TEST=$(curl \
```

Comment:
@kiwm, please add the `_` at the beginning of the variable `LATEST_TOMCAT_VERSION_TEST` name since it's private. I mean, it'll be used only inside this file.